fhir-path/src/main/java/com/ibm/fhir/path/function/MemberOfFunction.java

```diff
@@ -63,7 +67,7 @@ public int getMinArity() {
 
     @Override
     public int getMaxArity() {
-        return 1;
+        return 2;
```

Comment:
Please update the javadoc to indicate that we've added our own flavor of 'memberOf' which allows the caller to pass a binding strength

Reply:
Done

Reply:
new class doc looks good